microsoftgraph/microsoft-graph-explorer-v2
# Task/use hellojs

## Overview

Migrates authentication from using MSAL to using HelloJS

### Demo

N/A

### Notes

Access token is refreshed every hour.

## Testing Instructions

* Click on Sign In
* Observe that you're redirected
* Observe that the button label changes to Sign Out

Fixes [#49](https://github.com/microsoftgraph/microsoft-graph-explorer-v2/issues/49)
